src/depends/libDatabase/LevelDB.cpp

```diff
@@ -27,598 +26,548 @@
+  string value;
+  leveldb::Status s = m_db->Get(leveldb::ReadOptions(), key, &value);
+  if (!s.ok()) {
+    log_error(s);
+    return "";
+  }
+
+  return value;
 }
 
-string LevelDB::Lookup(const vector<unsigned char>& key) const
-{
-    string value;
-    leveldb::Status s = m_db->Get(leveldb::ReadOptions(), leveldb::Slice(vector_ref<const unsigned char>(&key[0], key.size())), &value);
-    if (!s.ok())
-    {
-        log_error(s);
-        return "";
-    }
+string LevelDB::Lookup(const vector<unsigned char>& key) const {
+  string value;
+  leveldb::Status s = m_db->Get(
+      leveldb::ReadOptions(),
+      leveldb::Slice(vector_ref<const unsigned char>(&key[0], key.size())),
+      &value);
+  if (!s.ok()) {
+    log_error(s);
+    return "";
+  }
 
-    return value;
+  return value;
 }
 
-string LevelDB::Lookup(const boost::multiprecision::uint256_t & blockNum) const
-{
-    string value;
-    leveldb::Status s = m_db->Get(leveldb::ReadOptions(), blockNum.convert_to<string>(), &value);
+string LevelDB::Lookup(const boost::multiprecision::uint256_t& blockNum) const {
+  string value;
+  leveldb::Status s =
+      m_db->Get(leveldb::ReadOptions(), blockNum.convert_to<string>(), &value);
 
-    if (!s.ok())
-    {
-        log_error(s);
-        return "";
-    }
+  if (!s.ok()) {
+    log_error(s);
+    return "";
+  }
 
-    return value;
+  return value;
 }
 
-string LevelDB::Lookup(const boost::multiprecision::uint256_t & blockNum, bool &found) const
-{
-    string value;
-    leveldb::Status s = m_db->Get(leveldb::ReadOptions(), blockNum.convert_to<string>(), &value);
-
-    if (!s.ok())
-    {
-        log_error(s);
-        found = false;
-        return "";
-    }
-    found = true;
-    return value;
-}
-
-string LevelDB::Lookup(const dev::h256 & key) const
-{
-    string value;
-    leveldb::Status s = m_db->Get(leveldb::ReadOptions(), leveldb::Slice(key.hex()), &value);
-    if (!s.ok())
-    {
-        log_error(s); 
-        return "";
-    }
+string LevelDB::Lookup(const boost::multiprecision::uint256_t& blockNum,
+                       bool& found) const {
+  string value;
+  leveldb::Status s =
+      m_db->Get(leveldb::ReadOptions(), blockNum.convert_to<string>(), &value);
 
-    return value;
+  if (!s.ok()) {
+    log_error(s);
+    found = false;
+    return "";
+  }
+  found = true;
+  return value;
 }
 
-string LevelDB::Lookup(const dev::zbytesConstRef & key) const
-{
-    string value;
-    leveldb::Status s = m_db->Get(leveldb::ReadOptions(), ldb::Slice((char const*)key.data(), 32),
-                                  &value);
-    if (!s.ok())
-    {
-        log_error(s);
-        return "";
-    }
+string LevelDB::Lookup(const dev::h256& key) const {
+  string value;
+  leveldb::Status s =
+      m_db->Get(leveldb::ReadOptions(), leveldb::Slice(key.hex()), &value);
+  if (value.empty()) {
+    LOG_GENERAL(WARNING, "BZ: There's no value for key: " << key.hex());
```

Comment:
Also add this log inside the `if (!s.ok())` - Maybe value could be not empty but `s` is an error?